### PR TITLE
Add zerone tool

### PR DESCRIPTION
**Pull request by mvdbeek**
Zerone is a neat little tool that can deal both with TF chipseq datasets and chromatin modifier datasets without needing to tweak parameters.

FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

FOR REVIEWER:
* [ ] .shed.yml file ok
    - [ ] Toolshed user `iuc` has access to associated toolshed repo(s)
* [ ] Indentation is correct (4 spaces)
* [ ] Tool version/build ok
* [ ] `<command/>`
  - [ ] Text parameters, input and output files `'single quoted'`
  - [ ] Use of `<![CDATA[ ... ]]>` tags
  - [ ] Parameters of type `text` or having `optional="true"` attribute are checked with `if str($param)` before being used
* [ ] Data parameters have a `format` attribute containing datatypes recognised by Galaxy
* [ ] Tests
  - [ ] Parameters are reasonably covered
  - [ ] Test files are appropriate
* [ ] Help
  - [ ] Valid restructuredText and uses `<![CDATA[ ... ]]>` tags
* [ ] Complies with other best practice in [Best Practices Doc](http://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html)

### CODE REVIEW

**Comment by mvdbeek** [comment]
(depends on https://github.com/bioconda/bioconda-recipes/pull/10814)

File: tools/zerone/zerone.xml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    <inputs>
+        <param argument="--chip" type="data" format="sam,bam" multiple="true" label="ChIP alignment file(s)"/>
+        <param argument="--mock" type="data" format="sam,bam" multiple="true" label="Control alignment file(s)"/>
+        <param argument="--window" type="integer" value="300" min="1" label="Window size in bp (default 300)"/>
+        <param argument="--quality" type="integer" value="20" min="1" label="Minimum mapping quality (default 20)"/>
+        <param argument="--list_output" type="boolean" truevalue="--list-output" falsevalue="" label="Output list of targets (default table)"/>
+        <param argument="--confidence" type="boolean" truevalue="--confidence" falsevalue="" label="Print targets only with higher confidence" help="Restricts intervals accordingly in list output"/>
+    </inputs>
+    <outputs>
+        <data name="output" format="tabular" label="Zerone on $on_string"/>
```
Comment (bgruening) [inline tools/zerone/zerone.xml]: This could be qualified as `interval` isn't it?

Comment (mvdbeek) [reply tools/zerone/zerone.xml]: Yeah, I think that'd work.

**Comment by mvdbeek** [comment]
Thanks @martenson!

**Pull request merged**